src/std-types/vec.md

```diff
@@ -20,6 +20,9 @@ fn main() {
     // Canonical macro to initialize a vector with elements.
     let mut v3 = vec![0, 0, 1, 2, 3, 4];
 
+    // specifying the type when initializing a vector.
+    let mut v4 = Vec::<String>::new();
```

Comment:
A more idiomatic way to do this would be
```rust
    let mut v4: Vec<String> = Vec::new();
```
which was covered in the type-inference slide on day 1.

The turbofish operator (`::<String>`) is introduced in the speaker notes in the impl trait slide in the "Generics" segment and further refined in the sections on FromIterator. I think that's enough coverage for the moment.

Reply:
> A more idiomatic way to do this would be
> 
> ```rust
>     let mut v4: Vec<String> = Vec::new();
> ```
> 
> which was covered in the type-inference slide on day 1.
> 
> The turbofish operator (`::<String>`) is introduced in the speaker notes in the impl trait slide in the "Generics" segment and further refined in the sections on FromIterator. I think that's enough coverage for the moment.

Oh! That's my bad i haven't  noticed it. Then lets just close the PR.

Reply:
What do you say.

Reply:
Sounds good!